lib/os.ml

```diff
@@ -147,3 +153,12 @@ let ensure_dir path =
   match check_dir path with
   | `Present -> ()
   | `Missing -> Unix.mkdir path 0o777
+
+let copy ?(superuser=false) ~src dst =
+  if Sys.win32 then
+    exec ["robocopy"; src; dst; "/MIR"; "/NFL"; "/NDL"; "/NJH"; "/NJS"; "/NC"; "/NS"; "/NP"]
```

Comment:
Not 100% familiar with this tool but it seems right based off https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy

Reply:
It's a bit arcane, but it's working without too much noise log.